.gitignore

```diff
@@ -15,3 +15,4 @@
 local.properties
 SecretRingKey.gpg
 /buildSrc/build/
+*.jks
```

Comment:
Interesting, curious what prompted this?

Reply:
While trying to crack the code on the GoPuff BT issue I had to create key files that now live in my root filesystem and I was hoping to keep them there for future use! If you don't like it I can always move them somewhere else and document the process on how to create them.

Reply:
Nope this is totally OK - was just curious!